app/controllers/api/v2/hosts_controller.rb

```diff
@@ -109,6 +109,7 @@ def show
           param :capabilities, String
           param :compute_profile_id, :number
           param :interfaces_attributes, Array, :desc => N_("Host's network interfaces.") do
+            param :id, :number, :desc => N_("ID of interface")
```

Comment:
Looking at the redmine issue, it seems this parameter is rejected? i think that to allow it you need to add it in the strong params definition or handle it correctly inside the update action, the params are only used to generate the api docs, so it shouldn't be blocking it.

Reply:
No, it's only rejected by API clients who actually read the apidoc. I might have phrased the redmine wrongly.

Reply:
can you please update the commit message to explain that then?